lacework/provider.go

```diff
@@ -59,6 +59,7 @@ func Provider() terraform.ResourceProvider {
 			"lacework_alert_channel_slack":           resourceLaceworkAlertChannelSlack(),
 			"lacework_alert_channel_splunk":          resourceLaceworkAlertChannelSplunk(),
 			"lacework_alert_channel_service_now":     resourceLaceworkAlertChannelServiceNow(),
+			"lacework_alert_channel_victor_ops":      resourceLaceworkAlertChannelVictorOps(),
```

Comment:
I think this might be better resource name.

https://github.com/splunk/terraform-provider-victorops
```suggestion
			"lacework_alert_channel_victorops":      resourceLaceworkAlertChannelVictorOps(),
```